src/main/java/com/rtb/projectmanagementtool/task/TaskController.java

```diff
@@ -67,14 +56,13 @@ public void addTasks(HashSet<TaskData> tasks) {
   public void deleteTasks(HashSet<Long> taskIDs) {
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     HashSet<Key> keys = new HashSet<>();
-    Query query = new Query("Comment").setKeysOnly();
+    Query query = new Query("Task");
+    Filter filter = new Query.FilterPredicate("__key__", Query.FilterOperator.IN, taskIDs);
+    query.setFilter(filter);
```

Comment:
I'm curious if we can take a list of taskIDs and build keys using the [KeyFactory](https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/KeyFactory) so we wouldn't have to query for the keys first before deleting.